src/lib/axl-explorations/src/components/games/LetterLauncherGameStoryPreview.tsx

```
@@ -1225,18 +1225,26 @@ export function LetterLauncherGameStoryPreview({
           visibility: isVisible ? 'visible' : 'hidden'
         }}
       >
-        <div className="relative w-full max-w-[123px] xs:max-w-[140px] sm:max-w-[220px] md:max-w-sm lg:max-w-md xl:max-w-lg">
+        <div className={`relative w-full ${
+          arrowAlign === 'right' 
+            ? 'max-w-[240px] sm:max-w-[220px] md:max-w-sm lg:max-w-md xl:max-w-lg' 
+            : 'max-w-[123px] xs:max-w-[140px] sm:max-w-[220px] md:max-w-sm lg:max-w-md xl:max-w-lg'
```

ESLint

[error] Replace `arrowAlign·===·'right'·⏎············?·'max-w-[240px]·sm:max-w-[220px]·md:max-w-sm·lg:max-w-md·xl:max-w-lg'·⏎············:·'max-w-[123px]·xs:max-w-[140px]·sm:max-w-[220px]·md:max-w-sm·lg:max-w-md·xl:max-w-lg'` with `··arrowAlign·===·"right"⏎··············?·"max-w-[240px]·sm:max-w-[220px]·md:max-w-sm·lg:max-w-md·xl:max-w-lg"⏎··············:·"max-w-[123px]·xs:max-w-[140px]·sm:max-w-[220px]·md:max-w-sm·lg:max-w-md·xl:max-w-lg"`

(prettier/prettier)